deploy.bash

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# No failures allowed
+set -e
+
+# The name of the preface for the docker images
+DOCKER_NAME="cnxta/cdr-"
+
+# The IP For the place you're storing this
+DEPLOY_IP=""
+
+# The port used by docker
+PORT=""
+
+# The stack you want the docker compose file named
+STACK="cdr-stack"
+
+# The command to build docker images
+BUILD_CMD="./gradlew clean build -x test"
+
+# The path to the Docker Wrapper
+DOCKER_W=""
+
+GITHUB_PATCH=""
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Tests that variables are correct for the docker compose and docker wrapper
+
+if [ -z ${DEPLOY_IP} ] && [ -z ${PORT} ] && [ -z ${DOCKER_W} ]; then
+    echo "Set the global variables at the top of $0"
```

Comment:
❓ Is there a way we can set these variables when we run the script so that I don't have to modify this file? Something like `--stackName ... --deployIp ...`.

Reply:
I think you can do that with  the Bash version of getopts
https://www.computerhope.com/unix/bash/getopts.htm

Reply:
I decided to make it so you can have either variables in your .bash_profile or edit the file. I don't think there's a need to accept parameters as not much is going to change between deployments